.github/workflows/import-linter.yml

```
@@ -69,38 +75,55 @@ jobs:
+                      echo "has_changes=true" >> "$GITHUB_OUTPUT"
                     fi
-
-                    {
-                      echo "has_changes=true"
-                      echo "files<<EOF"
-                      printf "%s\n" "${changed_py[@]}"
-                      echo "EOF"
-                    } >> "$GITHUB_OUTPUT"
+                    echo "Changed Python files detected: ${file_count}"
 
             -   name: Ruff (autoformat changed Python files)
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)